OpenSim/Common/DelimFileAdapter.h

```diff
@@ -47,11 +93,20 @@ class OSIMCOMMON_API DelimFileAdapter : public FileAdapter {
     DelimFileAdapter(const std::string& delimitersRead,
                      const std::string& delimterWrite);
 
+    /** Create the adapter by setting the delimiters.                         */
+    DelimFileAdapter(const std::string& delimitersRead,
+                     const std::string& delimterWrite,
+                     const std::string& compDelimRead,
+                     const std::string& compDelimWrite);
+
     DelimFileAdapter* clone() const override;
 
     /** Key used for table associative array returned/accepted by write/read. */
     static const std::string _table;
 
+    /** Name of the data type T (template parameter).                         */
+    static const std::string dataTypeName();
+
```

Comment:
I did. There needs to be *guarantee* that the string produces is going to be *exactly same*:
* Across platforms.
* Across compilers.
* Over time.

I think the current implementation does this.